content/news/known-amendments.md

```diff
@@ -5,13 +5,14 @@ The following is a comprehensive list of all known amendments and their status o
 
 | Name                      | Introduced | Status                              |
```

Comment:
Making the tables sortable would be a matter of (1) adding some javascript to the templates, and (2) maybe adding something to make sure the javascript applied to the correct tables. Possibly making every table in the content body is an option, in which case (2) wouldn't be that hard.

The table itself is sorted in what attempts to be "chronological" order for things that don't always have an obvious chronology. In practice, this means the table is sorted primarily by status, with secondary sorts depending on the status and a tertiary sort by name. The long descriptions outside the table are in alphabetical order.  The sorting criteria is complicated enough that I think describing it would be more distracting than useful.

Reply:
> The sorting criteria is complicated enough that I think describing it would be more distracting than useful.

I have to vehemently disagree with that reasoning. If it's so complicated that we can't even explain, that's a problem for users. It had been a while since I'd seen this page when reviewed it and it took me a lot more time than I would've liked to figure out where this amendment was. The double presentation of the list (in the page and side toc) and the idiosyncratic sorting did it not make it a very intuitive experience (somewhat the opposite, in fact). I looked to the description for some logic as to the sorting and didn't get anything, which increased my frustration. 

I know this is, overall, a fairly minor point but I don't think we should get in the habit of deferring or omitting explanations because something is too complicated. And, actually, "sorted in this order: status, chronologically (by release version), and alphabetically (by amendment name)" doesn't seem all that complicated.

Reply:
That's because the sort order is more complicated than that.

The [actual sort order](https://github.com/ripple/amendmentbot/blob/master/src/amendment.py#L56-L76) is:

1. Sort by status in the following order: (No status), "Planned", "Open for Voting", "Expected", "Enabled", "Vetoed"
    - (Aside: it's weird that "In Development" isn't in the AmendmentBot source there. Not sure how it's working without sorting those, then.)
2. Secondary sort depends on the status:
    - For "Planned", "Open for Voting", and "Vetoed" statuses, sort descending by introduced version. (slightly undesirably, this means "TBD" comes after "vX.Y.Z". That's probably something to change, actually)
    - For "Expected" status, sort descending by date expected (that is, later dates first)
    - For "Enabled" status, sort descending by date enabled (most recently enabled first)
3. Tertiary sort is ascending by name

I don't mind adding a line like, "sorted in chronological order (most recent first)" to the description, but digging that deeply into the details of the sorting implementation, especially when it's likely to change subtly, is completely besides the point of the document.

The document has two use cases:

1. You want to know what's currently going on with amendments. Did that amendment you need get enabled? What's in voting right now? What's coming soon?
    - The table of known amendments serves this need by listing the most recent amendments at the top, most active. If you scroll down through the table, you get essentially a history of prior changes in the order they occurred, with the failed potential changes at the bottom.
2. You want to know what a specific amendment does.
    - The "In this page" table, listed by chronological order, serves this need. Amendments don't move around within the page content after they're added, and regardless of what happened to the amendment you can find it by name.